renovate.json5

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      // Compose compiler is tightly coupled to Kotlin version
+      "groupName": "Kotlin and Compose",
+      "matchPackagePrefixes": [
+        "org.jetbrains.compose.compiler",
+        "org.jetbrains.kotlin",
```

That's a cool change! 👍

I noticed the project also has coroutines as a dependency; this 👇 would avoid grouping them together with Kotlin
```suggestion
        "org.jetbrains.kotlin:kotlin",
```

---

Ah, right. I always forget it's a simple string prefix and not an actual groupId match.